tests/TestCase/Model/Table/QueuedTasksTableTest.php

```diff
@@ -422,56 +423,56 @@ public function testRequestGroup() {
 		];
 
 		// create an ungrouped task
-		$this->assertTrue((bool)$this->QueuedTasks->createJob('task1', 1));
+		$this->assertTrue((bool)$this->QueuedTasksTable->createJob('task1', 1));
 		//create a Grouped Task
-		$this->assertTrue((bool)$this->QueuedTasks->createJob('task1', 2, null, 'testgroup'));
+		$this->assertTrue((bool)$this->QueuedTasksTable->createJob('task1', 2, null, 'testgroup'));
 
 		// Fetching without group should completely ignore the Group field.
-		$this->QueuedTasks->clearKey();
-		$tmp = $this->QueuedTasks->requestJob($capabilities);
```

Comment:
Changing all those was not necessary and only makes the diff harder to read. Please try to stick to the absolute minimum of changes.